pymc3/distributions/discrete.py

```diff
@@ -332,6 +339,19 @@ def logcdf(self, value):
         )
 
 
+class BernoulliLogitRV(BernoulliRV):
```

Comment:
Should it inherit from BernoulliRV or RandomVariable directly?

Is the `name` and `_print_name` change just confusing and not necessary?